es-ds-components/components/es-form-checkbox.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+const model = defineModel({
+    default: false,
+});
+defineOptions({
+  inheritAttrs: false
```

Comment:
nit: 2 spaces instead of 4

Reply:
Done via [43d3fbf](https://github.com/EnergySage/es-ds/pull/1485/commits/43d3fbfded02e77eeea20aec9712e54de7ebb828)